docs/core/classes.md

```diff
@@ -345,22 +308,20 @@ qx.Class.define("qx.test.Cat", {
   },
   members: {
     makeSound : function(x) {
-      this.constructor.someStaticMethod(x);
+      qx.test.Animal.someStaticMethod(x);
     }
   }
 });
 ```
 
 The syntax for accessing static variables simply is
-`this.constructor.someStaticVar`. Please note, for `this.constructor` to be
-available, the class must be a derived class of `qx.core.Object` , which is
-usually the case for regular, non-static classes.
-
-Instead of `this.constructor` you can also use the alternative syntax
-`this.self(arguments)`.
+`qx.test.Animal.someStaticVar`.  Please note that `this.constructor` is not
+recommended because it always refers to the subclass of the object, and there
+can change change the implementation at run time.  Always specify class names
```

Comment:
"and there can change change" - probably something wrong here?

Reply:
done in hkollmann_doc_v7